besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -707,6 +707,17 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final Integer engineRpcWsPort = DEFAULT_WEBSOCKET_ENGINE_PORT;
 
+  @Option(
+      names = {"--engine-jwt-secret"},
+      paramLabel = MANDATORY_FILE_FORMAT_HELP,
+      description = "Path to file containing shared secret key for JWT signature verification")
+  private final Path engineJwtKeyFile = null;
```

Comment:
Might be clearer to have HMAC/HS256 in the name as well since we have a lot of other auth secrets/pub keys for other things using other algorithms

Reply:
Engine JWT keys can only ever be HS256 per the spec, so I think that might be unnecessary.